.github/workflows/release.yml

```diff
@@ -6,6 +6,9 @@ on:
 jobs:
   release:
     runs-on: ubuntu-22.04
+    environment:
+      name: release
+      url: https://pypi.org/p/tox
```

Comment:
Can you explain this line? Why we need it?

Reply:
This will show up in UI, also you can apply protections in the repo settings, once it appears. I used to scope secrets to environments, instead of global settings. You can also require human approvals. All's optional.

Reply:
It's optional but is required when you fill out the field on the PyPI side.

Reply:
If you don't want the `url:` line, you can also just have `environment: release`.

Reply:
It's fine, but I'll need to create my own PR tomorrow because it doesn't let me merge this. Apparently you don't have the necessary rights to make this change. Or so GitHub says. ...

Reply:
Oh, are you using the GitHub (mobile?) app? That thing has a limited token IIRC — using a web UI should help in such a case.